src/test/resources/processor_test_files/2057_SerialVersionUidCheck/UpperClass.java

```diff
@@ -0,0 +1,4 @@
+import java.io.Serializable;
+
+class UpperClass implements Serializable {
```

Comment:
Assuming that Sonar flags the issue on the line of the class header (I've never used this particular check, so I'm not sure), this should solve it.
```suggestion
class UpperClass implements Serializable { // Noncompliant
```

Reply:
Yep, test output says this:

```
java.lang.AssertionError: Unexpected at [3]
```

Which in Sonar assertion error speak means "unexpected violation at line 3". We should catch and improve this error message to make it less confusing for new contributors.